client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -27,11 +47,171 @@ export default function Dashboard(): JSX.Element {
     return <CircularProgress />;
   }
 
+  //temp
+  const cards = [1, 2, 3, 4, 5, 6];
+  const loggedIn = true;
+
+  const handleMenuClick = (e: React.MouseEvent<HTMLButtonElement>) => {
+    console.log(e.currentTarget);
```

Comment:
Could you remove the console.log here, if your current PR isn't fully working then please make sure it works properly. PRs are for merging into main and it should all be working properly.